mmv1/third_party/terraform/tests/resource_cloud_run_v2_service_test.go

```diff
@@ -403,3 +480,121 @@ resource "google_cloud_run_v2_service" "default" {
 }
 `, context)
 }
+
+func testAccCloudRunV2Service_cloudRunServiceUpdateWithEmptyGRPCLivenessProbe(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_cloud_run_v2_service" "default" {
+  name     = "tf-test-cloudrun-service%{random_suffix}"
+  location = "us-central1"
+
+  template {
+    containers {
+      image = "us-docker.pkg.dev/magic-modules-374220/cloud-run-hello-grpc/cloud-run-hello-grpc"
```

Comment:
This gRPC server container image is hosted in my personal temporary repository.
As explained in [`mmv1/third_party/terraform/utils/test-fixtures/cloudrun/hello-cloud-run-grpc/README.md`](https://github.com/mmizutani/magic-modules/blob/cloud-run-v2-grpc-probe/mmv1/third_party/terraform/utils/test-fixtures/cloudrun/hello-cloud-run-grpc/README.md), this custom image is needed for testing the gRPC probe of the `google_cloud_run_v2_service` resource, but no official image of ready-to-use, prebuilt gRPC server container image for Cloud Run is provided by Google Cloud or gRPC schosystem developers.

It would be nice if we could host this container image in the same Google Cloud project as `gcr.io/graphite-docker-images/magic-modules-dev`. I am thinking of adding a GitHub Actions workflow like [`magic-modules/.ci/gcb-downstream-builder.yml`](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/gcb-downstream-builder.yml) that runs only when files under `mmv1/third_party/terraform/utils/test-fixtures/cloudrun/hello-cloud-run-grpc/` change in the main branch.

Otherwise, we need to build this container image during each run of the `google_cloud_run_v2_service` service, but we would end up with spending majority of the time for tests in testing this docker build workflow rather than in testing Cloud Run services.
Theoretically it could be possible to automate this in `make test`, but this requires adding the Go Docker client library and Docker/Podman runtime to CI dependencies.

Reply:
We definitely will not want to host this in a contributor project. Is there a publicly accessible image we can rely on? I don't like the idea of managing an application image for a single test.

Reply:
>  Is there a publicly accessible image we can rely on?

Unfortunately, no. 
I did search for publicly accessible container images of a gRPC server with the standard health check implementation, but I was not able to find any. Neither Google Cloud nor the gRPC ecosystem team provide such a convenient free image.

Reply:
Only the grpc startup probe, not the grpc liveness probe, requires an actually working gRPC server.
So I concluded that in the meantime I will postpone adding support for the grpc block in the startup probe in this PR.

Reply:
>  Is there a publicly accessible image we can rely on?

Unfortunately, no. 
I did search for publicly accessible container images of a gRPC server with the standard health check implementation, but I was not able to find any. Neither Google Cloud nor the gRPC ecosystem team provide such a convenient free image.

Only the grpc startup probe, not the grpc liveness probe, requires an actually working gRPC server.
So I concluded that in the meantime I will postpone adding support for the grpc block in the startup probe in this PR.

Reply:
Maybe we can use something like 
https://github.com/GoogleCloudPlatform/golang-samples/tree/f04b64bb5e595bcf7d3726a7eb7ed0d0557ed90a/run/grpc-ping#deploying-to-cloud-run 

Although this might have a prerequisite of building the image. The other alternative would be to use an image that doesn't work correctly and expect the deployment to fail. We would still be exercising the APIs acceptance of these fields which would be sufficient for our coverage.

Reply:
> The other alternative would be to use an image that doesn't work correctly and expect the deployment to fail. We would still be exercising the APIs acceptance of these fields which would be sufficient for our coverage.

I have followed this course of your advice and added this type of tests: 
https://github.com/mmizutani/magic-modules/blob/68ddd69ac609fc9c113af93cefca600c43d55685/mmv1/third_party/terraform/tests/resource_cloud_run_v2_service_test.go#L304-L328